spark-operator/src/test/java/org/apache/spark/k8s/operator/probe/ReadinessProbeTest.java

```diff
@@ -37,7 +37,6 @@
 
 import org.apache.spark.k8s.operator.utils.ProbeUtil;
 
-@SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
```

Comment:
We are using `UnitTestShouldIncludeAssert` instead of `JUnitTestsShouldIncludeAssert`. And, this is excluded in the config file already.